### DISCUSSION TITLE
I am login github today .But I don't find canva premium

### DISCUSSION THREAD

**Discussion by amerina576** [community/community#39276] [GitHub Education]
github don't offer canva pro for new student??  they have unveiled this service?

How Can I get canva pro??as i am a new student.

**Comment by dhrubasaha08** [comment]
Sorry to let you know that the complimentary Canva Pro membership offered via the GitHub Student Developer Pack has been discontinued. @amerina576

**Reply by amerina576** [reply]
Now, what is the offer of GitHub as designing software?

**Comment by dhrubasaha08** [comment]
@amerina576 They no longer provide any kind of design software. In the meanwhile, you may check out Icons8 IconScout included in the GitHub Student Developer Pack.
